inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/responses/ping/MessageMixerResponseSpec.kt

```diff
@@ -186,7 +186,8 @@ class MessageMixerResponseSpec(private val testname: String, private val actual:
                 messagePayload.messageSettings?.controlSettings)
         private val displaySettings = DisplaySettings(messageSettings.displaySettings?.orientation!!,
                 messageSettings.displaySettings.slideFrom, messageSettings.displaySettings.endTimeMillis,
-                messageSettings.displaySettings.textAlign, messageSettings.displaySettings.optOut)
+                messageSettings.displaySettings.textAlign, messageSettings.displaySettings.optOut,
+                messageSettings.displaySettings.delay)
         private val controlSettings = ControlSettings(messageSettings.controlSettings?.buttons,
```

Comment:
Is there a test for the delay behavior? If not, it'd be good to have one.

Reply:
added test cases with mock checks to know what value of delay was executed.

Reply:
For the actual delay behavior, it will be hard to test since it uses worker.
But sample app was tested with logged time/duration.